doc/util/chpl2md.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+"""chpl2md converts a chapel program to an md file, where all comments are
+rendered md, and all code is wrapped in code blocks.
+
+Chapel files are converted as follows:
+
+* Line comments starting at the beginning of the line are converted to text
+* Block comments starting anywhere are converted to text
+* All other lines are wrapped as code-blocks
+
+example.chpl
+============
+
+/* This is
+     an example */
+proc foo() {
+    // this comment will be in the code block
+    var bar = 1; }
+// this comment will be text
+
+example.rst
+===========
+
+This is
+  an example
+
+```chapel
+proc foo() {
+    // this comment will be in the code block
+    var bar = 1; }
+```
+
+this comment will be text"""
+
+from __future__ import print_function
```

Comment:
extraneous for python 3 I believe

Reply:
It was originally in `chpl2rst`, so I'm keeping it here. I can, of course, delete it, but my aim has been to minimize changes that aren't relevant to splitting out the code.